CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -813,15 +769,6 @@ static CFTimeZoneRef __CFTimeZoneCreateSystem(void) {
         CFRelease(name);
         if (result) return result;
     }
-#if DEPLOYMENT_TARGET_ANDROID
```

Comment:
I suspect most of the changes in `CFTimeZone.c` should probably prefer the SCL-F version.

Reply:
Yeah. This bit absolutely needs to be preserved.